package.nix

```diff
@@ -1,15 +1,16 @@
 {
   lib,
+  stdenv,
   python3,
   makeWrapper,
   e2fsprogs-nofortify,
   erofs-utils,
-  fetchFromGitHub,
```

Comment:
it was just added a commit before!?

I think the previous commit b91550e8e1e8d230915740af945b0cb12d8bd7a4 could be squashed into this one.

Reply:
Fixed by dropping the commit. It had nothing to do here.